addons/html_editor/static/src/core/overlay.js

```diff
@@ -75,13 +76,15 @@ export class EditorOverlay extends Component {
         }
         let rect = range.getBoundingClientRect();
         if (rect.x === 0 && rect.width === 0 && rect.height === 0) {
+            this.props.shared.disableObserver();
```

Comment:
Add a comment
```
Attention, using disableObserver and enableObserver is always dangerous (when we add or remove nodes) because if another mutation uses the target that is not observed, that mutation can never be applied again (when undo/redo and in collaboration).
```